gen-docs/main.go

```diff
@@ -20,8 +20,7 @@ updatedAt: %s
 title: "%s"
 slug: %s
 url: %s
----
-`
+---`
```

Comment:
@raulb I think this new line was needed here. If you look at the front matter it results in something like:

```
---
createdAt: 2021-04-15T10:19:35+02:00
updatedAt: 2021-04-15T10:19:35+02:00
title: "meroxa version"
slug: meroxa-version
url: /cli/meroxa-version/
---## meroxa version
```

Reply:
@anaptfox oops, my bad!